client/state/account-recovery/reset/reducer.js

```diff
@@ -17,23 +17,23 @@ import {
 const isRequesting = ( state = false, action ) => get( {
 	[ ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST ]: true,
 	[ ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE ]: false,
-	[ ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR ]: true,
+	[ ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR ]: false,
 }, action.type, state );
 
 const error = ( state = null, action ) => get( {
 	[ ACCOUNT_RECOVERY_RESET_OPTIONS_RECEIVE ]: null,
 	[ ACCOUNT_RECOVERY_RESET_OPTIONS_ERROR ]: action.error,
```

Comment:
Might want to clear the error out on a `ACCOUNT_RECOVERY_RESET_OPTIONS_REQUEST` too